starters/next-drupal-starter/README.md

```diff
@@ -1,18 +1,19 @@
-# Next Drupal Starter
-
-## Demo Next.js Project setup
+# Pantheon Decoupled Kit Next Drupal Starter
 
 1. Clone this repo:
-```
-git clone git@github.com:pantheon-systems/decoupled-drupal-frontend-demo.git
+
+```bash
+git clone git@github.com:pantheon-systems/next-drupal-starter.git
 ```
 
 2. Install node modules
-```
+
+```bash
 cd decoupled-drupal-frontend-demo && npm install
 ```
 
-3. update following lines in `.env.local`
+3. Update following lines in `.env.local`
```

Comment:
If / when the markdown docs have additional details about these envars, we should link to that here.